src/DotNet.Status.Web/Controllers/DeploymentController.cs

```diff
@@ -174,7 +174,7 @@ private async Task<CloudTable> GetCloudTable()
         private static readonly ImmutableArray<MediaTypeFormatter> s_grafanaFormatters =
             ImmutableArray.Create(s_grafanaFormatter);
 
-        private class AnnotationEntity : TableEntity
+        public class AnnotationEntity : TableEntity
```

Comment:
... public?  Other things really shouldn't reference an entire website, certainly not a nest class of a controller.  If this is something we want to share, it probably needs to move into another class in some other DLL.

Reply:
Good idea; will do